.travis.yml

```diff
@@ -22,6 +22,9 @@ before_install:
   - cat /etc/lsb-release
 
 install:
+  - pip install -U flake8
```

Comment:
Could you change to the `pip3 install -q -U -r ./tests/requirements.txt`?
It error causes only `--user`, and I want to CI test with hacking modules (more strictly)